eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -79,10 +86,7 @@ jobs:
         pip3 install azure.storage.queue==12.0.0 &&
         sudo apt-get update &&
         sudo apt -y install curl dirmngr apt-transport-https lsb-release ca-certificates &&
-        curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&
-        sudo apt-get -y install nodejs &&
-        npm install --prefix $HELIX_WORKITEM_PAYLOAD jsvu -g &&
-        $HELIX_WORKITEM_PAYLOAD/bin/jsvu --os=linux64 --engines=v8,javascriptcore &&
+        $(HelixPreCommandsWasmOnLinux) &&
```

Comment:
Does this evaluate to blank or to the string if the variable is empty? I can't remember. Secondly, wouldn't we want to include the `&&` in the create of `HelixPreCommandsWasmOnLinux` so that we don't end up with an extra `&&` lying around when we don't want it.

Reply:
Yeah, the blank thing might be a problem because the variable name is also under the condition. I'll fix that.

re:&&, I'll set the value to `echo` for non-wasm cases. Adding `&&` at the end of the command sounds wrong because it might, or might not be valid where it is being used